netci.groovy

```diff
@@ -276,6 +276,44 @@ def buildArchConfiguration = ['Debug': 'x86',
 }
 
 // **************************
+// Define outerloop testing for netfx that can build and run.  Run locally on each machine.
+// **************************
+[true, false].each { isPR ->
```

Comment:
Because it doesn't support all OS only Windows and the other outerloop runs are for all the OSs and netcoreapp specific. In order to add the netfx run and in windows I would have to add another .each with both netcoreapp and netfx targetgroups and then skip all the OSs but Windows_NT when the target group is netfx, so I thought adding it in a new group was cleaner.